### PR TITLE
fix(fmgc): nat track and lat/lon improvements

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #[issue_no]

## Summary of Changes

- Fix AWY/WPT MISMATCH when uplinking a route with NAT tracks
- Fix bug in lat/lon waypoint creation... probably never occurs anymore with the above fix though
- Add extended lat/lon waypoint format... common on A330/340, but maybe some people like it on A320

## Screenshots (if necessary)

Extended format:
image
Setting:
image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Import some NAT track routes from SimBrief and see your QoL improved. Try with

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/EFB/Settings/Settings.tsx [hunk 1]
```diff
@@ -157,6 +158,11 @@ const AircraftConfigurationPage = () => {
         { name: '25 kHz', setting: '1' },
     ];
 
+    const latLonExtendedButtons: ButtonType[] = [
+        { name: 'LLnn', setting: '0' },
+        { name: 'AxxByyy', setting: '1' },
```
[the remark this answers is not in the document]
Comment (tracernz) [reply src/instruments/src/EFB/Settings/Settings.tsx:163]: Maybe can put the actual format as tooltips in flyPad OSv3, but unfortunately that’s not merged yet.

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCDataManager.js [hunk 2]
```diff
@@ -18,6 +18,8 @@ class FMCDataManager {
 
         this.storedWaypoints = [];
 
+        this.latLonExtendedFormat = false;
```
[the remark this answers is not in the document]
Comment (tracernz) [reply flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCDataManager.js:21]: Hmmm, I think the original is more descriptive, which is more fitting for the basically global scope of the member.

File: src/instruments/src/EFB/Settings/Settings.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (2hwk) [reply src/instruments/src/EFB/Settings/Settings.tsx:163]: Alright I guess we can revisit this after

**Review by 2hwk — APPROVED** [review]

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7003
Tier of Testing  : 1
Date             : 13/04/2022

Testing Process: 
Import some NAT track routes from SimBrief and see your QoL improved. - **OK**

Negatives:       
I don't know if it's realistic but having NAT track as an airway (instead of wpt-wpt course) in the flight plan would be really nice.

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image
image
image

**Comment by tracernz** [comment]
> I don't know if it's realistic but having NAT track as an airway (instead of wpt-wpt course) in the flight plan would be really nice.

It would be but NAT tracks change every day so they can’t be in the nav database.

**Pull request merged**